pyuvdata/uvdata/tests/test_ms.py

```diff
@@ -1063,3 +1074,13 @@ def test_flip_conj_multispw(sma_mir, tmp_path):
     sma_mir.filename = ms_uv.filename = None
 
     assert sma_mir == ms_uv
+
+def test_read_ms_write_miriad(nrao_uv, nrao_uv_legacy, tmp_path):
```

Comment:
It looks like this is causing a name conflict here in the testing (which is also causing a linting failure) -- maybe rename this `test_read_ms_write_miriad_legacy`?